daemon/logger/fluentd/fluentd.go

```diff
@@ -153,7 +153,9 @@ func (f *fluentd) Log(msg *logger.Message) error {
 	}
 	// fluent-logger-golang buffers logs from failures and disconnections,
 	// and these are transferred again automatically.
-	return f.writer.PostWithTime(f.tag, msg.Timestamp, data)
+	err := f.writer.PostWithTime(f.tag, msg.Timestamp, data)
```

Comment:
I wonder if it makes sense to copy required fields and `PutMessage` before `PostWithTime`.
Probably not a big deal.